README.md

```diff
@@ -79,7 +79,7 @@ against are in the [`schemas/`](schemas) directory.
 
 ## Team
 
-GOV.UK Replatforming team looks after this repo. If you're inside GDS, you can find us in [#govuk-replatforming](https://gds.slack.com/archives/C013F737737) or view our [kanban board](https://trello.com/b/u4FCzm53/).
+[GOV.UK Platform Engineering](https://github.com/orgs/alphagov/teams/gov-uk-platform-engineering) team looks after this repo. If you're inside GDS, you can find us in [#govuk-platform-engineering] or view our [kanban board](https://trello.com/b/u4FCzm53/).
```

Comment:
think the slack channel still needs a link

Reply:
Whoops, thanks!

Reply:
Ah it was a reference-style link from further up. I moved the reference to the end to make it clearer.